imports/plugins/core/orders/client/components/OrderRefunds.js

```diff
@@ -0,0 +1,302 @@
+import React, { Fragment, useEffect, useRef, useState } from "react";
+import PropTypes from "prop-types";
+import { Mutation } from "react-apollo";
+import { Form } from "reacto-form";
+import withStyles from "@material-ui/core/styles/withStyles";
+import Card from "@material-ui/core/Card";
+import CardContent from "@material-ui/core/CardContent";
+import CardHeader from "@material-ui/core/CardHeader";
+import FormControl from "@material-ui/core/FormControl";
+import InputLabel from "@material-ui/core/InputLabel";
+import Grid from "@material-ui/core/Grid";
+import MenuItem from "@material-ui/core/MenuItem";
+import OutlinedInput from "@material-ui/core/OutlinedInput";
+import Select from "@material-ui/core/Select";
+import Typography from "@material-ui/core/Typography";
+import ErrorsBlock from "@reactioncommerce/components/ErrorsBlock/v1";
+import Field from "@reactioncommerce/components/Field/v1";
+import TextInput from "@reactioncommerce/components/TextInput/v1";
+import { i18next, Reaction } from "/client/api";
+import formatMoney from "/imports/utils/formatMoney";
+import ConfirmButton from "/imports/client/ui/components/ConfirmButton";
```

Comment:
Use https://catalyst.reactioncommerce.com/#/Components/Feedback/ConfirmDialog instead

Reply:
Does `ConfirmDialog` have the button built in?

`ConfirmButton` and `ConfirmDialog` were two different components in `reaction`.

Reply:
Which button? The first button is not built-in per say. You add it like this:

```
import Button from "../Button";

<ConfirmDialog
  title="Archive 24 products?"
  message="Archiving products removes them from both admin and customer views."
  onConfirm={() => { alert("Action confirmed!") }}
>
  {({ openDialog }) => (
    <Button color="primary" onClick={openDialog} variant="contained">Open Confirm Dialog</Button>
  )}
</ConfirmDialog>
```

Reply:
Ok... `ConfirmButton` is doing the same thing, just built in already so you didn't have to import `Button` as well.

Will update.

Reply:
Created a ticket to revisit: https://github.com/reactioncommerce/reaction/issues/5421